src/docs/arch-hs.dox

```diff
@@ -4,5 +4,111 @@ The History Store in WiredTiger tracks old (all but the latest committed)
 versions of records.  By having these records in storage separate from
 the current version, they can be used to service long running transactions,
 and be evicted as necessary, without interfering with activity that uses
-the most recent committed versions.
+the most recent committed versions. With introduction of History Store, only the
+newest update to a key is written to the user table while the older updates
+for the key are written to the History Store. All user tables in the database are
+backed up by a single History Store table. History Store has no outward
```

Comment:
I think you'd want to say "backed by" not "backed up" by as that could be misinterpreted as a backup?